addOns/jwt/jwt.gradle.kts

```
@@ -0,0 +1,21 @@
+import org.zaproxy.gradle.addon.AddOnStatus
+
+version = "1"
```

This may as well use semver from the start.
```suggestion
version = "1.0.0"
```